docs/tutorials/morpheusvm/5_interlude.md

```diff
@@ -6,118 +6,132 @@ local network and interacting with it via the HyperSDK-CLI.
 
 In this section, we'll:
 
-- Setting up our run/stop scripts
-- Adding end-to-end tests
+- Setting up the Avalanche-CLI
+- Defining our Genesis
 - Setting up our VM binary generator
 - Installing our CLI
 
 Let's get started!
 
-## Script Setup
+### Avalanche-CLI Installation
```

Comment:
Can we link to AvalancheCLI's installation instructions and name it as a pre-requisite rather than copy-pasting them here?

Reply:
Is interlude still an appropriate name for this section?

Since we no longer depend on the e2e test to setup a network, would the following order make sense:

- write action unit tests
- create and interact with a network via AvalancheCLI + HyperSDK universal CLI
- advanced section for integration/e2e tests

Reply:
Done